app/src/ui/create-tag/create-tag-dialog.tsx

```diff
@@ -67,6 +71,17 @@ export class CreateTag extends React.Component<
             initialValue={this.props.initialName}
             onValueChange={this.updateTagName}
           />
+
+          {this.state.previousTags && (
```

Comment:
Nit, we try to avoid falsy/truthy checks
```suggestion
          {this.state.previousTags !== null && (
```